rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/source/Querier.java

```diff
@@ -0,0 +1,220 @@
+
+        log.trace("Creating a PreparedStatement '{}'", SQL);
+        PreparedStatement stmt = db.prepareStatement(SQL);
+        return stmt;
+
+    }
+
+    protected PreparedStatement createPreparedStatement(Connection db, String string) throws SQLException {
+        String query = "select * from " + string;
+        log.trace("Creating a PreparedStatement '{}'", query);
+        PreparedStatement stmt = db.prepareStatement(query);
+        return stmt;
+
+    }
+
+    protected ResultSet executeQuery(PreparedStatement stmt) throws SQLException {
+        return stmt.executeQuery();
+    }
+
+    private Schema schema;
+
+    public static void main(String[] args) throws Exception {
+        TimestampIncrementingQuerier querier = new TimestampIncrementingQuerier();
+        try {
+            querier.start();
+            querier.poll();
+        } catch (SQLException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+    }
+
+    public void poll() {
+        try {
+
+            PreparedStatement stmt;
+            String query = "select * from ";
+            Connection conn = dataSource.getConnection();
+            for (Map.Entry<String, Database> entry : schema.dbMap.entrySet()) {
+                String db = entry.getKey();
+                Iterator<Map.Entry<String, Table>> iterator = entry.getValue().tableMap.entrySet().iterator();
+                while (iterator.hasNext()) {
+                    Map.Entry<String, Table> tableEntry = iterator.next();
+                    String tb = tableEntry.getKey();
+                    stmt = conn.prepareStatement(query + db + "." + tb);
+                    ResultSet rs;
+                    rs = stmt.executeQuery();
+                    List<String> colList = tableEntry.getValue().getColList();
+                    List<String> DataTypeList = tableEntry.getValue().getRawDataTypeList();
+                    List<ColumnParser> ParserList = tableEntry.getValue().getParserList();
+
+                    while (rs.next()) {
+                        Table table = new Table(db, tb);
+                        System.out.print("|");
+                        table.setColList(colList);
+                        table.setRawDataTypeList(DataTypeList);
+                        table.setParserList(ParserList);
+
+                        for (String string : colList) {
+                            table.getDataList().add(rs.getObject(string));
+                            System.out.print(string + " : " + rs.getObject(string) + "|");
+                        }
+                        list.add(table);
+                        System.out.println();
+                    }
+                }
+            }
+
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    public void start() throws Exception {
+        try {
+
+            log.info("datasorce success");
+            initDataSource();
```

Comment:
if `dataSource` init error ,maybe stop task is correct way, if still run task `schema.load()` will throw NPE